_posts/payalpatel/2016-04-05-PayalsCulturalImmersionActivity_PearlHacks.md

```diff
@@ -0,0 +1,15 @@
+
+The sponsor fair followed the opening ceremony.  At the sponsor fair, there were several companies represented there including Cisco, Google, Fidelity, and CapitalOne.  The sponsor fair was a great opportunity to talk to representatives from different companies to learn more about them and the types of positions they have.  (They also gave out a lot of free stuff, which is always a plus. :) )   
+
+Pearl Hacks also had several workshops offered during the afternoon.  The first workshop I attended was called “Beginner Tutorial: Mobile Development (Android/Android Studio)”.  This workshop was by a first-year college student from Georgia Tech and an IT employee from Fidelity Investments.  I think this workshop had a lot of potential; however, most of the time was used to get people to download Android Studio.  I think it would have been more beneficial if they had informed people ahead of time of things they would need to download/install for participating in this workshop.  Overall, I think this workshop was a good experience because it got me thinking about building and creating apps.  I learned a little about IDE (integrated development environments) and SDK (software development kits).  I started researching ways I could create apps with the knowledge I have.  This workshop focused on Android Apps and using Java/Android Studio.  After researching a little online I found that there are other tools available to create apps and some of these tools allow you to create apps that can be used on multiple platforms (iOS, Android, and Windows), such as Visual Studio or Eclipse.  Here’s a link I found for a post that talks about Python IDE’s : https://opensource.com/business/15/10/top-open-source-python-ides.  In the end, with this workshop I didn’t get a lot of hands-on experience; however, it did inspire me to think of ideas and learn how to create an app myself.  This workshop has motivated me to push myself to try and create an app myself.  
```

Comment:
Sometimes organization/coordination is lacking in volunteer-led events.  Installing is a bear. Still, glad if gave you some ideas!